pkg/controller/dtversion/docker_hashes.go

```diff
@@ -1,26 +1,25 @@
-package version
+package dtversion
 
 import (
 	"context"
 	"fmt"
 	"strings"
 
-	"github.com/Dynatrace/dynatrace-operator/pkg/controller/parser"
 	"github.com/containers/image/v5/manifest"
 	"github.com/containers/image/v5/transports/alltransports"
 	"github.com/containers/image/v5/types"
 	"github.com/opencontainers/go-digest"
 )
 
-// Deprecated: DockerLabelsChecker implements a preferred version check using image labels
+// Deprecated: DockerLabelsChecker implements a preferred version check using image matchLabels
 type DockerHashesChecker struct {
 	currentImage   string
 	currentImageId string
-	dockerConfig   *parser.DockerConfig
+	dockerConfig   *DockerConfig
 }
 
-// Deprecated: DockerLabelsChecker implements a preferred version check using image labels
-func NewDockerHashesChecker(currentImage, currentImageId string, dockerConfig *parser.DockerConfig) *DockerHashesChecker {
+// Deprecated: DockerLabelsChecker implements a preferred version check using image matchLabels
+func NewDockerHashesChecker(currentImage, currentImageId string, dockerConfig *DockerConfig) *DockerHashesChecker {
```

Comment:
Never used?

Reply:
We decided to deprecate the old logic instead of deleting it